scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -283,11 +283,11 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
     {% set display_genes = [] %}
     {% for gene in variant.genes %}
       {% if gene.hgvs_identifier and gene.hgnc_symbol  %}
-        {{ "" if display_genes.append(gene.hgnc_symbol + ' ' + gene.hgvs_identifier) }}
+        {{ "" if display_genes.append(gene.hgnc_symbol + ' ' + gene.hgvs_identifier|truncate(20,True)) }}
       {% elif gene.hgnc_symbol %}
           {{ "" if display_genes.append(gene.hgnc_symbol) }}
       {% elif gene.hgvs_identifier and gene.hgnc_id %}
-          {{ "" if display_genes.append( gene.hgnc_id|string + ' ' + gene.hgvs_identifier) }}
+          {{ "" if display_genes.append( gene.hgnc_id|string + ' ' + gene.hgvs_identifier|truncate(20,True)) }}
```

Comment:
Good! I was thinking of the version we have on the somatic variant page, https://github.com/Clinical-Genomics/scout/blob/035544c4dc5bef1b8b60f9474fe80abf04b3c84a/scout/server/blueprints/variant/templates/variant/cancer-variant.html#L253, with a truncated plus tooltip. But, these are clickable more often than not, and perhaps used in constructions where a direct tooltip would be complicated. Lets go with this - we can extend it later if needed.